docs/native/src/scenarios/operational_flows.rst

```diff
@@ -0,0 +1,67 @@
+..  _operational flows:
+
+Operational flows
+=================
+
+This section outlines the implementation requirements and operational flows for user scenarios outlined in :ref:`Browsing, opening and saving files<browse files>` section. 
+
+**Terminology**
```

Comment:
Just make this a [definition list](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#definition-lists). You shouldn't format this stuff yourself.